plumber/parse.go

```diff
@@ -0,0 +1,274 @@
+package plumber
```

Comment:
Really great that this has been moved. So much cleaner.

Reply:
🎉